cache/build.gradle.kts

```diff
@@ -90,14 +82,6 @@ mavenPublishing {
     signAllPublications()
 }
 
-addGithubPackagesRepository()
-kmmbridge {
```

Comment:
Thanks for opening @aclassen! What is the reason for removing KMM Bridge? Our CI uses KMM Bridge to generate a Swift package on release

Reply:
So the idea is to use store direct as pod in objc ?

Reply:
We want to make `cache` and `store` available for iOS folks

Reply:
So store should be available as a regular pod  , without any shared code ?

Reply:
Not sure I am understanding - What do you mean by "shared code"?

Reply:
I don't have an opinion on `multicast` but think it makes sense to make `cache` available as a Swift package. Without any question we need to make `store` available as one.

Reply:
What i mean is, do you want to be able to the following : open xcode -> create a new ios project -> add store as pod in your podfile and use it in swift.

Reply:
Yes

Reply:
oh ok

Reply:
Thanks 🙏🏽

Reply:
@matt-ramotar Can you maybe help me to fix the failing workflow ?

Reply:
Mind adding `iosSimulatorArm64` in a separate PR? We will need to modify our CI and I unfortunately won't be able to work on this until next week sometime. See #511